### PR TITLE
Feature/viirs metadata

### PR DESCRIPTION
**Pull request by solomon-negusse**
## Pull request checklist

Please check if your PR fulfills the following requirements:
- [x] Make sure you are requesting to pull a topic/feature/bugfix branch (right side). Don't request your master!
- [x] Make sure you are making a pull request against the develop branch (left side). Also you should start your branch off our develop.
- [x] Check the commit's or even all commits' message styles matches our requested structure.
- [x] Check your code additions will fail neither code linting checks nor unit test.

## Pull request type

 

 

Please check the type of change your PR introduces:
- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe): 

## What is the current behavior?

Issue Number: [GTC-2390](https://gfw.atlassian.net/browse/GTC-2390)
Fire alerts don't send data content date range

## What is the new behavior?

Fire alerts updates send content end date

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

## Other information

[GTC-2390]: https://gfw.atlassian.net/browse/GTC-2390?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/datapump/jobs/version_update.py [hunk 1]
```diff
@@ -137,14 +137,6 @@ def _create_aux_tile_set(self, tile_set_parameters: RasterTileSetParameters) ->
                 "num_processes": co.num_processes,
                 "resampling": co.resampling,
             },
-            "metadata": {
```
Comment (jterry64) [inline src/datapump/jobs/version_update.py:140]: why was this removed?

**Review by jterry64 — CHANGES_REQUESTED** [review]
I think you need to add some more to complete it, and also it'd be good to try testing a sync in staging after we merge this (or even just pushing this dev code to staging to test).

File: src/datapump/jobs/version_update.py (hunk 1 above)
Comment (solomon-negusse) [reply src/datapump/jobs/version_update.py:140]: This was a hotfix for the new metadata schema that I just pulled from `master`.  This was removed because it was specifying version level metadata for assets.

File: src/Dockerfile [hunk 2]
```diff
@@ -11,7 +11,7 @@ COPY . $WORKDIR
 # installing dependencies to build package
 RUN pip install . -t python
 
-# change 31860
```
[the remark this answers is not in the document]
Comment (solomon-negusse) [reply src/Dockerfile:14]: No, really just scoped this PR for GTC-2390 issue of fire alerts updates and considered it won't be a big deal to add start date manually since the tables are not created frequently.
What you mention and adding the metadata for all tables occurred to me but need to dig more into the code.

**Comment by solomon-negusse** [comment]
> I think you need to add some more to complete it, and also it'd be good to try testing a sync in staging after we merge this (or even just pushing this dev code to staging to test).

Thanks @jterry64, will make the update for creation and test sync which I was planning on doing.

**Comment by solomon-negusse** [comment]
@jterry64:  `content_date_range` is now updated via `PATCH` request to the version endpoint only if appends succeed.
The corresponding API [PR](https://github.com/wri/gfw-data-api/pull/391) reverts updates to /append endpoint that updated version `content_date_range`.

**Review by jterry64 — APPROVED** [review]
Looks good to me, thanks for clarifying those things!

**Review by dmannarino — APPROVED** [review]
